DOCSP-40216 Add search index management limitations compass (#634)
mongodb/docs-compass: source/indexes.txt

- The :guilabel:`Indexes` tab is not available if you are connected 
  to a :atlas:`Data Lake </data-lake>`. 

> - You can only manage Atlas Search indexes in Compass if your deployment is

Reviewer: (nit -- [positive statements](https://www.mongodb.com/docs/meta/style-guide/writing/use-positive-statements/))
You could omit "only" here if you like. The sentence will mean the same thing, but is more positive.

Author: Thanks for the edit, just fixed!

  local, has an Atlas cluster tier of M10 or larger, and runs MongoDB 7.0 
  or higher. 
  
  .. tip::

     For clusters running an earlier version of MongoDB or 
     :ref:`shared tier clusters <shared-tier-cluster>`, 

>      you can manage your Atlas Search indexes from the Atlas UI, with the 
>      :ref:`Atlas CLI <atlas-cli>`, or with the 
>      :ref:`Atlas Administration API <atlas-admin-api>`. 

Reviewer: (nit -- wording)
This is subjective and optional. Consider a slight tweak to create more parallelism. 
```suggestion
     you can manage your Atlas Search indexes using the Atlas UI, the 
     :ref:`Atlas CLI <atlas-cli>`, or the 
     :ref:`Atlas Administration API <atlas-admin-api>`. 
```

Author: Love parallelism, changed!
